server/db/seeds/users.rb

```diff
@@ -0,0 +1,23 @@
+connection = ActiveRecord::Base.connection()
+@adminRole = connection.execute("SELECT id FROM roles WHERE \"name\"='Admin'")
+User.create(
+  password: ENV["USER_PASSWORD"],
+  email: ENV["USER_EMAIL"],
+  nick_name: "seedUser",
+  password_confirmation: ENV["USER_PASSWORD"]
+)
+User.create(
+  password: ENV["ADMIN_PASSWORD"],
+  email: ENV["ADMIN_EMAIL"],
+  nick_name: "admin",
+  password_confirmation: ENV["ADMIN_PASSWORD"]
+)
+comands = [
+  "UPDATE \"users\" SET \"confirmed_at\"='2022-03-22 17:19:40.342358' WHERE \"nick_name\"='seedUser';",
+  "UPDATE \"users\" SET \"confirmed_at\"='2022-03-22 17:19:40.342358' WHERE \"nick_name\"='admin';",
```

Comment:
why not `User.where(nick_name: 'admin').first&.update(confirmed_at: 5.days.ago)` ?